### ISSUE TITLE
DNN/TF(EfficientNet): add Fill layer, ONNX: add Split layer default axis + const-propagation

### ISSUE THREAD

**Issue by teaglin** [opencv/opencv#19696]
##### System information (version)

- OpenCV => 4.5.1
- Operating System / Platform => Mac OS
- Compiler => Xcode

##### Detailed description

Using efficient lite model trained in Keras, then exported through tf crashes upon load using DNN. I've tried several different avenues. tf1 / tf2/ onnx. None seem to work. I've included 2 attempts through tensor flow. Both have different crashes.

##### 
**Attempt 1.**

`.1/modules/dnn/src/tensorflow/tf_importer.cpp:591: error: (-2:Unspecified error) Input [batch_normalization_1/cond/ones_like] for node [batch_normalization_1/cond/FusedBatchNorm] not found in function 'getConstBlob'
`
Empty model can be accessed here.
https://www.dropbox.com/s/jfpisyrqokqxagl/frozen_graph.pb?dl=0

##### 
**Attempt 2.**

`/opencv/opencv-4.5.1/modules/dnn/src/dnn.cpp:614: error: (-2:Unspecified error) Can't create layer "model_2/global_max_pooling2d_1/Max" of type "Max" in function 'getLayerInstance'
`
Empty model can be accessed here.
https://www.dropbox.com/s/rxl1ng4h6gcn0y0/frozen_graph2.pb?dl=0

##### 
**Attempt 3.**

This was done using tf2Onnx. So converting directly from Keras to Onnx.

```
[ERROR:0] global /opencv/opencv-4.5.1/modules/dnn/src/onnx/onnx_importer.cpp (1878) handleNode DNN/ONNX: ERROR during processing node with 3 inputs and 1 outputs: [Conv]:(model_2/efficientnet-b2/stem_bn/FusedBatchNormV3:0)
libc++abi.dylib: terminating with uncaught exception of type cv::Exception: OpenCV(4.5.1) opencv/opencv-4.5.1/modules/dnn/src/onnx/onnx_importer.cpp:1887: error: (-2:Unspecified error) in function 'handleNode'
> Node [Conv]:(model_2/efficientnet-b2/stem_bn/FusedBatchNormV3:0) parse error: OpenCV(4.5.1) opencv/opencv-4.5.1/modules/dnn/src/layers/convolution_layer.cpp:96: error: (-213:The function/feature is not implemented) Unsupported asymmetric padding in convolution layer in function 'BaseConvolutionLayerImpl'
> 
terminating with uncaught exception of type cv::Exception: OpenCV(4.5.1) opencv/opencv-4.5.1/modules/dnn/src/onnx/onnx_importer.cpp:1887: error: (-2:Unspecified error) in function 'handleNode'
> Node [Conv]:(model_2/efficientnet-b2/stem_bn/FusedBatchNormV3:0) parse error: OpenCV(4.5.1) /opencv/opencv-4.5.1/modules/dnn/src/layers/convolution_layer.cpp:96: error: (-213:The function/feature is not implemented) Unsupported asymmetric padding in convolution layer in function 'BaseConvolutionLayerImpl'
>

```
Empty model can be accessed here.
https://www.dropbox.com/s/03l9myisk15zfye/model.onnx?dl=0

**Comment by teaglin** [comment]
Any update on this? It's basically a show stopper at the moment and I can't find a workaround.

**Comment by alalek** [comment]
Please re-check issue with latest OpenCV code from 3.4 or master branch (which includes #20394 fix)

**Comment by teaglin** [comment]
Thanks, will try this out in next few days.

**Comment by asmorkalov** [comment]
@teaglin Friendly reminder.

**Comment by rogday** [comment]
@teaglin, the ONNX model you provided has 2 inputs, as opposed to TF version which has only one - is this intentional? `model_2/batch_normalization_1/FusedBatchNormV3`'s `scale` field is another input, which looks strange.

**Comment by teaglin** [comment]
Sorry haven't had a chance to try this out yet. Been swamped, working my way to it. @rogday that must be an issue with ONNX converter then. The model only has 1 input. A single, 3 - channel image.

**Comment by rogday** [comment]
Can you provide your original keras model in .h5 format?

**Comment by alalek** [comment]
Please re-check issue with latest OpenCV code from 3.4 or master branch

**Comment by abhishek-gola** [comment]
PR #28522 resolves the issue caused by the missing _BatchNormalization_ layer support in _ONNX_. With this change applied, the model now loads successfully in **OpenCV 5.x**.
